clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -31,27 +41,37 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
       <div id="fides-banner">
         <div id="fides-banner-inner">
           <CloseButton ariaLabel="Close banner" onClick={onClose} />
-          <div id="fides-banner-heading">
-            <div id="fides-banner-title" className="fides-banner-title">
-              {experience.title}
-            </div>
-            {showGpcBadge ? (
-              <GpcBadge
-                label="Global Privacy Control Signal"
-                status="detected"
-              />
-            ) : null}
-          </div>
           <div
-            id="fides-banner-description"
-            className="fides-banner-description"
+            id="fides-banner-inner-container"
+            style={{
+              gridTemplateColumns: children ? "1fr 1fr" : "1fr",
```

Comment:
If children are passed into this component then it means we've passing the vendor details component for the TCF banner. This means we have to adjust the grid to be two columns `1fr 1fr1` so the vendor details can be off to the right.